File: stepik-java-api/src/main/java/org/stepik/api/actions/StepikRecommendationReactionsAction.java
package org.stepik.api.actions;

import org.jetbrains.annotations.NotNull;
import org.stepik.api.client.StepikApiClient;
import org.stepik.api.queries.recommendations.StepikReactionsPostQuery;

/**
 * @author meanmail
 */
public class StepikRecommendationReactionsAction extends StepikAbstractAction {
    public StepikRecommendationReactionsAction(@NotNull StepikApiClient stepikApiClient) {
        super(stepikApiClient);
    }

    @NotNull
    public StepikReactionsPostQuery post() {
        return new StepikReactionsPostQuery(this);
    }

}

File: stepik-java-api/src/main/kotlin/org/stepik/api/actions/StepikRecommendationReactionsAction.kt
package org.stepik.api.actions

import org.stepik.api.client.StepikApiClient
import org.stepik.api.queries.recommendations.StepikReactionsPostQuery

class StepikRecommendationReactionsAction(stepikApiClient: StepikApiClient) : StepikAbstractAction(stepikApiClient) {
    
    fun post(): StepikReactionsPostQuery {
        return StepikReactionsPostQuery(this)
    }
    
}
